subprojects/docs/src/docs/userguide/gradle_wrapper.adoc

```diff
@@ -242,9 +242,11 @@ Gradle will report a build failure in case the configured checksum does not matc
 [[wrapper_checksum_verification]]
 == Verifying the checksum of the Gradle Wrapper JAR
 
-Since the Wrapper JAR file is usually checked in to a project's version control system, verifying its checksum provides a simple way to check its integrity.
-In addition, as with all binary artifacts, it's a way to ensure that it has not been tampered with, for example when reviewing a pull request that upgrades the Wrapper JAR in an open source project.
-The checksum of the Wrapper JAR is published alongside the corresponding Gradle distribution on https://services.gradle.org/.
+The Wrapper JAR is an executable binary file that is usually checked in to a project's version control system.
+As such, there is the potential for a malicious actor to replace the original JAR with a modified one thereby injecting arbitrary code that will be executed on the computers of developers and build servers.
```

Comment:
Do you mean that a malicious actor could directly replace the JAR that's in the version control system? Or that they could manipulate someone into downloading a malicious JAR which is then added to VCS?